src/pages/wishlist/main/Main.jsx

```diff
@@ -0,0 +1,37 @@
+import { wishlists } from "./dump";
+
+const Main = () => (
+  <>
+    <section class="margin-0 container">
+      <main class="sub-container">
+        {wishlists.map((wishlist) => (
+          <>
+            <section class="padding-0 card horizontal">
+              <img class="img" src={wishlist} alt="wishlist-products" />
+
+              <section class="header">
+                <h2 class="brand">Brand Name</h2>
+                <h3 class="price">Brand Price</h3>
+              </section>
+
+              <p class="text">
+                <strong>Product Name</strong>
+                and product information
+              </p>
+
+              <section class="card-footer">
+                <div class="icon">
+                  <a class="favourite" href="#">
+                    <i class="fas fa-shopping-cart"></i>Add to Cart
+                  </a>
+                </div>
+              </section>
+            </section>
```

Comment:
u can make a card component out of it and call it here would help in better readability